apps/web/src/routes/FullPageModal/FullPageModal.js

```diff
@@ -20,8 +20,8 @@ export default function FullPageModal ({
               key={tab.path}
             />
           ))}
-          {!multipleTabs && (content || children)}
         </Routes>
+        {!multipleTabs && (content || children)}
```

Comment:
In All Topics case, there are not multiple tabs, and thus, it was trying to render `children`, which was a div. And <Routes> only accepts <Route> as a valid child and it was flipping out